Application/Items/Commands/Create.cs

```diff
@@ -60,18 +53,21 @@ public async Task<Unit> Handle(Command request, CancellationToken cancellationTo
                 }
 
                 var user = await _context.Users.FindAsync(request.User);
-                request.Item.Uploader = user.Username;
+                
+                // gonna leave this as temp until front is connected to back
+                request.Item.Uploader = "temp";
```

Comment:
this is a temporary thing, until the front is connected to the back. will change back later